src/pages/iou/steps/MoneyRequestConfirmPage.js

```diff
@@ -141,17 +163,25 @@ function MoneyRequestConfirmPage(props) {
                 return;
             }
 
-            IOU.requestMoney(
-                props.report,
-                props.iou.amount,
-                props.iou.currency,
-                props.currentUserPersonalDetails.login,
-                props.currentUserPersonalDetails.accountID,
-                selectedParticipants[0],
-                trimmedComment,
-            );
+            if (props.iou.receiptPath && props.iou.receiptSource) {
+                FileUtils.readFileAsync(props.iou.receiptPath, props.iou.receiptSource).then((receipt) => {
```

Comment:
✋ Coming from https://github.com/Expensify/App/issues/28347

The transaction creation with receipt upload won't work after uploading the receipt and refreshing the page because the file is removed. So we decided to check the file, if the file is not available then we navigated back the user to the main request money page.